### PR TITLE
Changes for running bead maze experiments

### PR DESCRIPTION
**Pull request by carolinahiguera**
- Include `tactile_ssl` as observation encoder (tested for MAE).  It should work for all models, changing the encoder parameters in `train_diffusion_unet_beadmaze_image_workspace.yaml`
- The checkpoint for the encoders has to be a `.pth` with only the state_dict for each model. These are available  [here](https://drive.google.com/drive/u/0/folders/1SV1yS0mKv0H9rm3Q40v3K0_MZ-npy7qI)
- Action space corresponds to delta joint positions for the franka ONLY. We can include to the action space allegro joints later, after the first test.
- Observation space corresponds to representations for thumb and index `tactile_embed_dim + robot_joint_position + allegro_joint_position`
- It is ready for doing a first training to check hyperparameters. For running:
`python train.py --config-name=train_diffusion_unet_beadmaze_image_workspace training.seed=42 hydra.run.dir='data/outputs/${now:%Y.%m.%d}/${now:%H.%M.%S}_${name}_${task_name}`

- Dataset has to be in ./data/ in the root of the repo.
- Dataset in zarr format cab be downloaded from [here](https://drive.google.com/drive/u/0/folders/18kvArgi89QqXCehmdPl7fWQrK5Fz7iGh)

### CODE REVIEW

File: diffusion_policy/common/sampler.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ def __init__(self,
         if episode_mask is None:
             episode_mask = np.ones(episode_ends.shape, dtype=bool)
 
+        sequence_length = sequence_length + 7
```
Comment (akashsharma02) [inline diffusion_policy/common/sampler.py:101]: This is fine for now, but we should consider generalizing this for the future before code release

File: diffusion_policy/config/train_diffusion_unet_real_image_workspace.yaml [hunk 2]
```diff
@@ -106,7 +106,7 @@ training:
   gradient_accumulate_every: 1
   # EMA destroys performance when used with BatchNorm
   # replace BatchNorm with GroupNorm.
-  use_ema: True
+  use_ema: False
```
Comment (akashsharma02) [inline diffusion_policy/config/train_diffusion_unet_real_image_workspace.yaml:109]: Why are we changing this?

**Review by akashsharma02 — APPROVED** [review]
